specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/ipAddressManager.json

```diff
@@ -116,7 +116,7 @@
             "description": "IP Address Manager Pool resource name.",
             "required": true,
             "type": "string",
-            "pattern": "^[a-zA-Z0-9-]*$"
+            "pattern": "^[0-9a-zA-Z]([0-9a-zA-Z_.-]{0,62}[0-9a-zA-Z_])?$"
```

Comment:
Here and everywhere else.
What happens to the resources that fall under the previous regex and do not follow the new regex pattern ?